server/src/main/java/org/elasticsearch/common/settings/LocallyMountedSecrets.java

```diff
@@ -170,6 +170,13 @@ public byte[] getSHA256Digest(String setting) throws GeneralSecurityException {
         return MessageDigests.sha256().digest(getString(setting).toString().getBytes(StandardCharsets.UTF_8));
     }
 
+    /**
+     * @return version number from the secrets file
```

Comment:
nit: no need for `@return`, not sure how that would look without a summary. I would just say "Returns..."